Sources/ProjectDescription/SchemeDiagnosticsOption.swift

```diff
@@ -4,6 +4,15 @@ import Foundation
 
 /// Options to configure scheme diagnostics for run and test actions.
 public enum SchemeDiagnosticsOption: String, Equatable, Codable {
+    /// Enable the address sanitizer
+    case enableAddressSanitizer
+    
+    /// Enable the detect use of stack after return of address sanitizer
+    case enableASanStackUseAfterReturn
```

Comment:
Perhaps we can leverage a better name here - this doesn't need to map 1-1 with the underlying xcscheme names

 ```suggestion
    case enableDetectStackUseAfterReturn
```